src/backend/joanie/edx_imports/edx_factories.py

```diff
@@ -205,7 +205,7 @@ class Meta:
     id = factory.Sequence(lambda n: n)
     username = factory.Sequence(lambda n: f"{faker.user_name()}{n}")
     password = factory.Faker("password")
-    email = factory.Faker("email")
+    email = factory.sequence(lambda n: f"{faker.user_name()}{n}@example.com")
```

Comment:
It could be a lazy attribute in this way we could reuse the username, WDYT ?
```suggestion
    email = factory.Sequence(lambda n: f"{faker.user_name()}{n}@example.com")
```